packages/systemd-257/systemd-257.spec

```diff
@@ -0,0 +1,799 @@
+
+BuildRequires: gperf
+BuildRequires: intltool
+BuildRequires: meson
+BuildRequires: %{_cross_os}glibc-devel
+BuildRequires: %{_cross_os}kmod-devel
+BuildRequires: %{_cross_os}libacl-devel
+BuildRequires: %{_cross_os}libattr-devel
+BuildRequires: %{_cross_os}libblkid-devel
+BuildRequires: %{_cross_os}libcap-devel
+BuildRequires: %{_cross_os}libfdisk-devel
+BuildRequires: %{_cross_os}libmount-devel
+BuildRequires: %{_cross_os}libseccomp-devel
+BuildRequires: %{_cross_os}libselinux-devel
+BuildRequires: %{_cross_os}libuuid-devel
+BuildRequires: %{_cross_os}libxcrypt-devel
+
+Requires: %{_cross_os}kmod
+Requires: %{_cross_os}libacl
+Requires: %{_cross_os}libattr
+Requires: %{_cross_os}libblkid
+Requires: %{_cross_os}libcap
+Requires: %{_cross_os}libfdisk
+Requires: %{_cross_os}libmount
+Requires: %{_cross_os}libseccomp
+Requires: %{_cross_os}libselinux
+Requires: %{_cross_os}libuuid
+Requires: %{_cross_os}libxcrypt
+
+Provides: %{_cross_os}systemd = %{package_priority_epoch}:
+Conflicts: %{_cross_os}systemd
+
+%description
+%{summary}.
+
+%package console
+Summary: Files for console login using the System and Service Manager
+Requires: %{name}
+Provides: %{_cross_os}systemd-console = %{package_priority_epoch}:
+
+%description console
+%{summary}.
+
+%package cryptsetup
+Summary: Files for cryptsetup support in systemd
+Requires: %{name}
+Requires: %{_cross_os}cryptsetup
+Provides: %{_cross_os}systemd-cryptsetup = %{package_priority_epoch}:
+
+%description cryptsetup
+%{summary}.
+
+%package devel
+Summary: Files for development using the System and Service Manager
+Requires: %{name}
+Provides: %{_cross_os}systemd-devel = %{package_priority_epoch}:
+
+%description devel
+%{summary}.
+
+%package networkd
+Summary: Files for networkd
+Requires: %{name}
+Provides: %{_cross_os}systemd-networkd = %{package_priority_epoch}:
+
+%description networkd
+%{summary}.
+
+%package resolved
+Summary: Files for resolved
+Requires: %{name}
+Provides: %{_cross_os}systemd-resolved = %{package_priority_epoch}:
+
+%description resolved
+%{summary}.
+
+%prep
+%autosetup -n systemd-%{version} -p1
+
+%build
+CONFIGURE_OPTS=(
+ -Dmode=release
+
+ -Dsplit-bin=true
+ -Dlink-udev-shared=true
+ -Dlink-systemctl-shared=true
+ -Dlink-networkd-shared=false
+ -Dlink-timesyncd-shared=false
+ -Dlink-boot-shared=false
+ -Dstatic-libsystemd=false
+ -Dstatic-libudev=false
+
+ -Dsysvinit-path=''
+ -Dsysvrcnd-path=''
+ -Dinitrd=false
+ -Dnscd=false
+
+ -Dutmp=false
+ -Dhibernate=false
+ -Dldconfig=true
+ -Dresolve=true
+ -Defi=true
+ -Dtpm=false
+ -Denvironment-d=false
+ -Dbinfmt=true
+ -Drepart=enabled
+ -Dcoredump=false
+ -Dpstore=true
+ -Doomd=false
+ -Dlogind=true
+ -Dhostnamed=false
+ -Dlocaled=false
+ -Dmachined=false
+ -Dportabled=false
+ -Dsysext=false
+ -Dsysupdate=disabled
+ -Duserdb=false
+ -Dhomed=disabled
+ -Dnetworkd=true
+ -Dtimedated=false
+ -Dtimesyncd=false
+ -Dremote=disabled
+ -Dnss-myhostname=false
+ -Dnss-mymachines=disabled
+ -Dnss-resolve=enabled
```

Comment:
We don't use the NSS resolver plugin now - it caused a behavior change with Go DNS resolution - so this can likely be disabled.